File: pgm/ABC_Java/ABC_194_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final int solid15 = 15;
        final int solid10 = 10;
        final int solid3 = 3;
        final int fat8 = 8;
        final int fat3 = 3;
        final int one = 1;
        final int two = 2;
        final int three = 3;
        final int four = 4;
        final int zero = 0;
        final int A = Integer.parseInt (arr[zero]);
        final int B = Integer.parseInt (arr[one]);
        final int AB = A + B;
        if (solid15 <= AB && fat8 <= B) {
            System.out.println (one);
        } else if (solid10 <= AB && fat3 <= B) {
            System.out.println (two);
        } else if (solid3 <= AB) {
            System.out.println (three);
        } else {
            System.out.println (four);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_194_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.math.BigDecimal
// import java.math.RoundingMode
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer
// import java.util.TreeMap

// import kotlin.math.*

const val space: String = " "
const val solid15:Int = 15
const val solid10:Int = 10
const val solid3:Int = 3
const val fat8:Int = 8
const val fat3:Int = 3
const val one:Int = 1
const val two:Int = 2
const val three:Int = 3
const val four:Int = 4

fun main() {
    val (A: Int, B: Int) = readln().split(space).map { it.toInt() }
    val AB:Int = A + B
    if (solid15 <= AB && fat8 <= B) {
        println(one)
    } else if (solid10 <= AB && fat3 <= B) {
        println(two)
    } else if (solid3 <= AB) {
        println(three)
    } else {
        println(four)
    }
    kotlin.system.exitProcess(0)
}